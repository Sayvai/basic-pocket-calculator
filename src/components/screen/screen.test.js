import React from 'react';
import { render, screen } from '@testing-library/react';
import Screen from './screen';

describe('Screen component', () => {
  let props;

  beforeEach(() => {
    props = {
      output: '',
    };
  });

  it(`should render
      - the wrapper
      - and the input element with the value set via props.output`
    , () => {
      // given
      props.output = '7*7';

      // when
      const { container } = render(<Screen {...props} />);

      // then]
      expect(container).toMatchSnapshot();
    });
});