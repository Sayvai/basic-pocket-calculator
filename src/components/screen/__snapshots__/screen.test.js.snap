// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screen component should render
      - the wrapper
      - and the input element with the value set via props.output 1`] = `
<div>
  <div
    class="screen"
    data-testid="component-screen"
  >
    <input
      class="output"
      data-testid="component-screen-input"
      readonly=""
      value="7*7"
    />
  </div>
</div>
`;
